src/Form/RulesSettingsForm.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Form\RulesSettingsForm.
+ */
+
+namespace Drupal\rules\Form;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Psr\Log\LogLevel;
+
+/**
+ * Provides rules settings form.
+ */
+class RulesSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'rules_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['rules.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('rules.settings');
+
+    $form['log'] = array(
+      '#type' => 'checkbox',
+      '#title' => $this->t('Log debug information'),
```

Comment:
Can we use the description from the Drupal 7 version? "Log debug information to the system log". Rules will always be logging, it is just a question whether it will write that to the system log or not.